docs/spec/historical.rst

```diff
@@ -188,25 +188,30 @@ When checking Python 2.7 code, type checkers should treat the ``int`` and
 type ``str`` as well as ``unicode`` should be acceptable.
 
 
-Positional-only arguments
--------------------------
+Positional-only parameters
+--------------------------
 
 Some functions are designed to take their arguments only positionally,
 and expect their callers never to use the argument's name to provide
 that argument by keyword. Before Python 3.8 (:pep:`570`), Python did
-not provide a way to declare positional-only arguments.
+not provide a way to declare positional-only parameters.
```

Comment:
Nit: Do we really want to draw the distinction between arguments and parameters here? It's a little odd to see the paragraph switch from "argument" to "parameter" suddenly. I believe I saw a proposal once suggesting to use "arguments" for functions, and reserve "parameters" for generic types (and type forms)?

Reply:
Arguments refers to the values you pass to a function when you call it, whereas parameters defined as part of a function signature and are assigned the arguments at call time. I don't know what "positional-only arguments" would mean, so I find this phrase completely confusing. [PEP 570](https://peps.python.org/pep-0570/) consistently (and correctly, IMO) uses the phrase "positional-only parameters". That's also the phrase we use elsewhere in the typing spec. I don't think we should deviate from that phrase.